Tribler/Core/defaults.py

```diff
@@ -251,6 +252,7 @@
 tribler_defaults['Tribler']['window_height'] = 670
 tribler_defaults['Tribler']['sash_position'] = -185
 tribler_defaults['Tribler']['family_filter'] = 1
+tribler_defaults['Tribler']['monochrome_tray_icon'] = False
```

Comment:
Thanks for feedback. I made the changes. Not sure whether it's fine, but changes on icon are made after Tribler restart.